### DISCUSSION TITLE
Android TV App

### DISCUSSION THREAD

**Discussion by CoRe-za** [f1multiviewer/issue-tracker#279] [Feature requests] [answered]
This app is insane, it would be incredible to have ease of access on a TV

**Comment by f1multiviewer** [comment, marked as answer]
Hey, that's unfortunately way out of scope for this project, but there's other projects out there that might be sufficient for you, such as [Race Control TV](https://github.com/leonardoxh/race-control-tv).

**Reply by CoRe-za** [reply]
Thanks for the suggestion, Unfortunately that app doesn't compare to yours. Will see what I can figure out.  Appreciate your work and quick response.

**Reply by f1multiviewer** [reply]
Yeah I can see that, though MultiViewer likely requires way more resources than a TV (or even NVIDIA shield) has available at a time, but it's not my area of expertise, unfortunately

**Reply by CoRe-za** [reply]
Makes perfect sense. Thanks anyway. I will fiddle with casting or something.
